chopsticks-dstream/src/main/scala/dev/chopsticks/dstream/DstreamStateMetrics.scala

```diff
@@ -0,0 +1,10 @@
+package dev.chopsticks.dstream
+
+import dev.chopsticks.metric.{MetricCounter, MetricGauge}
+
+trait DstreamStateMetrics {
```

Comment:
Let's also have Factory and Manager for this Metrics trait as usual

Reply:
Done